sdk/src/wallet/message_interface/message_handler.rs

```diff
@@ -389,9 +395,15 @@ impl WalletMessageHandler {
         backup_path: PathBuf,
         stronghold_password: String,
         ignore_if_coin_type_mismatch: Option<bool>,
+        ignore_if_bech32_mismatch: Option<&str>,
```

Comment:
Why do we need backup/restore_backup as function here? They just call the real one it feels useless?

Reply:
It is useless, removed that already in the new bindings